index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+        <div class="all">
+            <div class="top-container">
+                <div class="header">
+                    <div class="logo"><a href="index.html" class="logo">My Logo</a></div>
+                    <div class="menu">
+                        <nav>
+                            <a href="index.html"><img src="image/Enabled.png" alt="menu"/></a>
+                        </nav>
+                    </div>
+                </div>
+                <section class="bg">
+                    <h1 class="name">I’m Eric Tuyishimire Glad to see you!</h1>
+                    <p class="content">
+                        I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                        <p class="connect">LET'S CONNECT</p>
+                        <ul class="social-1">
+                            <li><a href="index.html"><img src="image/1.png" alt="social1"/></a></li>
+                            <li><a href="index.html"><img src="image/2.png" alt="social2"/></a></li>
+                            <li><a href="index.html"><img src="image/3.png" alt="social3"/></a></li>
+                            <li><a href="index.html"><img src="image/4.png" alt="social4"/></a></li>
+                            <li><a href="index.html"><img src="image/5.png" alt="social5"/></a></li>
+                        </ul>
```

Comment:
- [ ] Kindly use `flex` display to list the social media icons as requested in the project requirement